user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/SampleInformationMain.java

```diff
@@ -281,8 +281,7 @@ private void onEditBatchClicked(EditBatchEvent editBatchEvent) {
   }
 
   private boolean isSampleRemovable(SampleId sampleId) {
-    ProjectId projectId = context.projectId().orElseThrow();
-    return deletionService.isSampleRemovable(sampleId, projectId);
+    return deletionService.isSampleRemovable(sampleId);
```

Comment:
Why was this touched in this PR? Removing a measurement should not remove a sample 😕

Reply:
Sorry about that. It turns out openBIS does not need the projectID for a lot of operations and I removed the clutter.